docker/jenkins/rstudio-version.sh

```diff
@@ -59,7 +59,12 @@ if [[ $DEBUG = false ]]; then
     EXTRA_CP_ARGS=--quiet
 fi
 
-MAJOR_VERSION="$(date -u '+%Y-%m')"
+if [ -e "RELEASE_VERSION" ]; then
+    MAJOR_VERSION="$(cat RELEASE_VERSION | tr -d ' ')"
```

Comment:
You could also use `tr -d '[:space:]'` here, to delete things like a potential trailing newline.

Reply:
```suggestion
    MAJOR_VERSION="$(cat RELEASE_VERSION | tr -d '[:space:]')"
```